public/apps/account/log-out-button.tsx

```diff
@@ -17,28 +17,39 @@ import React from 'react';
 import { EuiButtonEmpty } from '@elastic/eui';
 import { HttpStart } from 'opensearch-dashboards/public';
 import { logout, samlLogout } from './utils';
+import { AuthType, OPENID_AUTH_LOGOUT } from '../../../common';
 
 export function LogoutButton(props: {
-  authType: string;
+  authType: string[];
```

Comment:
Is this variable type change support backward compatible? Could you please test this case:
Create an existing 2.3 or older cluster that has a `string` type setting(only support single auth type) -> Upgrade the cluster to the 2.4 that has all the changes in this PR -> Do not change any settings -> See if it still works?

Reply:
For our existing design, we are not backwards compatible. When user upgrade to 2.4 version, they need to update the auth.type attribute with the new format  in opensearch_dashboards.yml.
- Multiple Auth Example: 
`opensearch_security.auth.type: ['basicauth', 'openid','saml']`
- Single Auth Example: 
`opensearch_security.auth.type: ['basicauth']`

If auth.type attribute is not updated, the following error messages will inform user to configure the opensearch_dashboards.yml properly:
` FATAL  Error: [config validation of [opensearch_security].auth.type]: could not parse array value from json input`

Reply:
agree with Chang, we need to keep backward compatible. consider:
```
enabled: schema.oneOf([schema.arrayOf(schema.string()), schema.string()]),
```

Reply:
Addressed in the new commit.